server/notification.js

```diff
@@ -135,9 +135,9 @@ class Notification {
      * @returns {Promise<string>} Successful msg
      * @throws Error with fail msg
      */
-    static async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+    static async send(notification, msg, monitorJSON = null, heartbeatJSON = null, downTime = null) {
```

Comment:
I think this change should be added to the `heartbeatJSON` as well.
What is the rationale behind adding another parameter?

If adding it to `heartbeatJSON` is not an option / adding a parameter this is better, please add this parameter to every function invocation to make this obvious to the authors of the notificationproviders)